src/ckeditor.ts

```diff
@@ -7,14 +7,15 @@
 
 import { defineComponent, h, markRaw, type PropType } from 'vue';
 import { debounce } from 'lodash-es';
-import type { EditorConfig } from '@ckeditor/ckeditor5-core';
-import type ClassicEditor from '@ckeditor/ckeditor5-build-classic';
+import type { Editor, DataApi, EditorConfig } from '@ckeditor/ckeditor5-core';
 
 const SAMPLE_READ_ONLY_LOCK_ID = 'Integration Sample';
 const INPUT_EVENT_DEBOUNCE_WAIT = 300;
 
+type EditorInstance = Editor & DataApi;
```

Comment:
This is not an instance.

Reply:
It is. On [line 53](https://github.com/ckeditor/ckeditor5-vue/pull/242/files#diff-74828324f821342a77b9371c3a3a5b78b4332754171cac0c835b609980bb2c5eR53) there is a param that accepts Editor class and returns a instance

Reply:
I believe we can use `Editor` without `DataApi` and refactor the current implementation to use `editor.data.get()` instead of `editor.getData()`.

Reply:
This is great suggestion! I updated the PR.